ansible/roles/operator-pipeline/tasks/community-release-pipeline-trigger.yml

```diff
@@ -75,9 +84,9 @@
                     git_pull_request_url: $(tt.params.git_pr_url)
                 spec:
                   timeouts:
-                    pipeline: "1h30m0s"
+                    pipeline: "4h15m0s"
```

Comment:
Do we need such timeout also for community? There are no additional steps in the ISV when used for community, so the old timing should be OK from my pov.